training/model_management/environments/model-management/environment.yaml

```
@@ -0,0 +1,9 @@
+image:
+  name: model-management
```

```suggestion
  name: azureml/curated/model-management
```

Also will need publishing section and mcr PR. https://github.com/microsoft/mcr/blob/main/teams/azureml/base/curated-public.yml

---

Done